modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/info/CommerceOrderItemInfoItemFields.java

```diff
@@ -0,0 +1,244 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.order.content.web.internal.info;
+
+import com.liferay.commerce.product.model.CPAttachmentFileEntry;
+import com.liferay.info.field.InfoField;
+import com.liferay.info.field.type.DateInfoFieldType;
+import com.liferay.info.field.type.NumberInfoFieldType;
+import com.liferay.info.field.type.TextInfoFieldType;
+import com.liferay.info.localized.InfoLocalizedValue;
+
+/**
+ * @author Alessio Antonio Rendina
+ */
+public class CommerceOrderItemInfoItemFields {
+
+	public static final InfoField<NumberInfoFieldType> companyIdInfoField =
+		BuilderHolder._builder.infoFieldType(
+			NumberInfoFieldType.INSTANCE
+		).name(
+			"companyId"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CommerceOrderItemInfoItemFields.class, "company-id")
+		).build();
+	public static final InfoField<DateInfoFieldType> createDateInfoField =
+		BuilderHolder._builder.infoFieldType(
+			DateInfoFieldType.INSTANCE
+		).name(
+			"createDate"
+		).labelInfoLocalizedValue(
+			InfoLocalizedValue.localize(
+				CommerceOrderItemInfoItemFields.class, "create-date")
+		).build();
+	public static final InfoField<TextInfoFieldType> decimalQuantityInfoField =
+		BuilderHolder._builder.infoFieldType(
+			TextInfoFieldType.INSTANCE
+		).name(
+			"decimalQuantity"
```

Comment:
we should take notes that this will need to be update after
https://liferay.atlassian.net/browse/COMMERCE-11743

Reply:
That's fine, but we are actually removing the field "decimalQuantity" and keeping only "quantity", so we will need to remove it from here also.